src/govuk/components/notification-banner/template.test.js

```diff
@@ -0,0 +1,276 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-env jest */
+
+const axe = require('../../../../lib/axe-helper')
+
+const { render, getExamples } = require('../../../../lib/jest-helpers')
+
+const examples = getExamples('notification-banner')
+
+describe('Notification-banner', () => {
+  describe('default example', () => {
+    it('passes accessibility tests', async () => {
+      const $ = render('notification-banner', examples.default)
```

Comment:
If we wanted to save rendering the same template multiple times (and remove some duplication), we could render the example once in a `beforeAll` function for the block, and then just make the assertions in each test.

(Also applies to some of the other describe blocks in this file)

Reply:
Chatted to @36degrees about this offline, it doesn't apply here so going to resolve comment.